src/Sylius/Bundle/ApiBundle/spec/Validator/Constraints/ShopUserResetPasswordTokenExistsValidatorSpec.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace spec\Sylius\Bundle\ApiBundle\Validator\Constraints;
+
+use PhpSpec\ObjectBehavior;
+use Sylius\Bundle\ApiBundle\Validator\Constraints\ShopUserResetPasswordTokenExists;
+use Sylius\Component\User\Model\UserInterface;
+use Sylius\Component\User\Repository\UserRepositoryInterface;
+use Symfony\Component\Validator\Constraint;
+use Symfony\Component\Validator\ConstraintValidatorInterface;
+use Symfony\Component\Validator\Context\ExecutionContextInterface;
+
+final class ShopUserResetPasswordTokenExistsValidatorSpec extends ObjectBehavior
+{
+    function let(UserRepositoryInterface $userRepository): void
+    {
+        $this->beConstructedWith($userRepository);
+    }
+
+    function it_is_a_constraint_validator(): void
+    {
+        $this->shouldImplement(ConstraintValidatorInterface::class);
+    }
+
+    function it_throws_an_exception_if_value_is_not_a_string(): void
+    {
+        $this
+            ->shouldThrow(\InvalidArgumentException::class)
+            ->during('validate', [null, new class() extends Constraint {
+            }])
```

Comment:
```suggestion
            ->during('validate', [null, new ShopUserResetPasswordTokenExists()])
```
same case as in `ShopUserResetPasswordTokenNotExpiredValidatorSpec`